src/main/java/com/diplomaProject/StudyBe/User/User.java

```diff
@@ -19,24 +19,22 @@ public class User implements UserDetails {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @Column(name = "first_name")
-    private String first_name;
-    @Column(name = "second_name")
-    private String last_name;
-    @Column(name = "email")
+    @Column(name = "username")
+    private String username;
+    @Column(name = "email", unique = true)
     private String email;
     @Column(name = "password")
     private String password;
 
     @ManyToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
     @JoinTable(
-            name = "users_roles",
+            name = "user_tags",
             joinColumns = @JoinColumn(
                     name = "user_id", referencedColumnName = "id"),
             inverseJoinColumns = @JoinColumn(
-                    name = "role_id", referencedColumnName = "id" )
+                    name = "subject_id", referencedColumnName = "id" )
     )
-    @Column(name = "roles")
+//    @Column(name = "subject")
```

Comment:
I do not see any migrations make sure to implement them before you make too many change to the database. They will help you keep track of the database state.